### ISSUE TITLE
Auto switch Language direction

### ISSUE THREAD

**Issue by IanSeng** [eddieantonio/predictive-text-studio#160]
# Background 
Currently user is allowed to manually toggle the language direction [#55](https://github.com/eddieantonio/predictive-text-studio/issues/55). Enhance the feature by adding the support automate the switching direction based on the language preference from the user. 

- In spike and do [#159]( https://github.com/eddieantonio/predictive-text-studio/pull/159), has proved that the Keyman api service returning object does include the language suggestion, hence this is a feasible solution to automate the language direction.

**Comment by eddieantonio** [comment]
So if I understand this correctly, we can automatically figure out if it's an LTR or RTL script using the Keyman API?

**Comment by IanSeng** [comment]
> So if I understand this correctly, we can automatically figure out if it's an LTR or RTL script using the Keyman API?

Yup, that is absolutely correct, Keyman Api does return a field **"isRTL": true** to indicate if a script is RTL or LTR
